work-with-us/index.html

```diff
@@ -6,11 +6,11 @@
 
 <section>
   <div class='container typography'>
-    <h1>Work with us</h1>
+    <h2>Work with us</h2>
 
     <p>Not ready to <a href='mailto:contact@apsis.io'>contact us</a> for an estimate? Here are some of the best reasons why you should choose Apsis Labs.</p>
```

Comment:
Instead of: Here are some of the best reasons why you should choose Apsis Labs

We should simplify to: Here are some reasons to work with us: